.github/workflows/deploy.yml

```
@@ -27,103 +31,162 @@ jobs:
+        echo "FAILED=$FAILED" >> $GITHUB_ENV
 
-        echo "SUCCESS_SERVICES=$SUCCESS_SERVICES" >> $GITHUB_ENV
-        echo "FAILED_SERVICES=$FAILED_SERVICES" >> $GITHUB_ENV
-
-    - name: Slack Notification (Success)
-      if: success()
+    - name: Slack Notification
       uses: slackapi/slack-github-action@v1.24.0
       with:
         payload: |
           {
```

actionlint (1.7.4)

unexpected token "[" while parsing variable access, function call, null, bool, int, float or string. expecting "IDENT", "(", "INTEGER", "FLOAT", "STRING"

(expression)